src/index.ts

```diff
@@ -274,9 +274,7 @@ async function catchupAndWatchChain(
       {
         toBlock: config.toBlock,
         logger: indexerLogger,
-        eventCacheDirectory: config.cacheDir
-          ? path.join(config.cacheDir, "events")
-          : null,
+        eventCacheDirectory: null,
```

Comment:
this change seems to have been checked in by accident? but it's the same in main so should be ok to merge

Reply:
Yes, it's unrelated to the change, I don't remember how it made its way here.